ios/Classes/QRView.swift

```diff
@@ -15,8 +15,11 @@ public class QRView:NSObject,FlutterPlatformView {
     var channel: FlutterMethodChannel
     var cameraFacing: MTBCamera
     
+    // Codabar, maxicode, rss14 & rssexpanded not supported. Replaced with qr.
+    // UPCa uses ean13 object.
     var QRCodeTypes = [
           0: AVMetadataObject.ObjectType.aztec,
+          2: AVMetadataObject.ObjectType.qr,
           2: AVMetadataObject.ObjectType.code39,
```

Comment:
Are there supposed to be multiple values for `2` here?

Reply:
Thank you, i made a typo!